src/partials-js/weather_buttons.js

```diff
@@ -0,0 +1,26 @@
+//const $todaySection = document.querySelector('.weather-container');
+//const $5DaysSection = document.querySelector('.Days5');
+//const $section_active = document.querySelector('.section-active');
+//const $section_not_active = document.querySelector('.section-not-active');
+
+export function setActive() {
+// Get the container element
+var btnContainer = document.getElementById("weather-buttons-container");
+
+// Get all buttons with class="days-btn" inside the container
+var btns = btnContainer.getElementsByClassName("days-btn");
+
+// Loop through the buttons and add the active class to the current/clicked button
+for (var i = 0; i < btns.length; i++) {
+  btns[i].addEventListener("click", function() {
```

Comment:
jak przekazujesz funkcje i chcesz aby sie wykonała dopiero po kliknieciu to przekazujesz referencje do funkcji czyli nazwa funkcji bez nawiasów.
a co do funkcj jaka przekazujesz to czy przekazanie funkcji która sie właśnie wykonuje działa dobrze ?
Wydaje mi się że pętla która dodaje addEventListener z wywołaniem funkcji nie powinna być w tej funkcji ale to musze jeszcze sprawdzic